commit 9fcd477efaa975f1b76f9ff5be1b950488afb3cd (refs/changes/81/681/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-13T14:40:13-07:00 (3 years ago)
    
    IMGAPI-591 updates.jo can't update manifests of images migrated in CM-712
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/bin/imgapi-standalone-restore b/bin/imgapi-standalone-restore
index a2bfe29..ba960e4 100755
--- a/bin/imgapi-standalone-restore
+++ b/bin/imgapi-standalone-restore
@@ -156,6 +156,7 @@ elif [[ $infoStatus != "200" ]]; then
 else
     echo "Restore /data/imgapi/images"
     $MANTASYNC -r $bakDir/images /data/imgapi/images
+    chown -R nobody:nobody /data/imgapi/images
 fi
 
 echo ""
@@ -168,6 +169,7 @@ elif [[ $infoStatus != "200" ]]; then
 else
     echo "Restore /data/imgapi/manifests"
     $MANTASYNC -r $bakDir/manifests /data/imgapi/manifests
+    chown -R nobody:nobody /data/imgapi/manifests
 fi
 
 # The restored.marker file is used to tell 'imgapi-standalone-backup' that it
